tests/integration/test_802.1x_srv_files/hostapd.eap_user

```diff
@@ -1 +1,2 @@
-* TLS
+*   PEAP,TLS
+"client.example.org"	MSCHAPV2	"password"	[2]
```

Comment:
There is no new line of this file.

Reply:
UNIX is expecting a \n at the end of file.

Reply:
Resolved